forms-web-app/src/views/appellant-submission/appeal-statement.njk

```diff
@@ -147,6 +147,7 @@
           name: "does-not-include-sensitive-information",
```

Comment:
Should we note put uploadedFile.id instead of name in the "Added file" part ?  
        {% if appeal.yourAppealSection.appealStatement.uploadedFile.name %}

Reply:
yeah, can do - gunna do this as a separate PR though, because I'm sick of this PR

Reply:
Don't worry, I'll do it on mine, it was more to have your confirmation